### ISSUE TITLE
load stops that correspond to the trips that are part of the specified service_id

### ISSUE THREAD

**Issue by d3netxer** [remix/partridge#67]
* partridge version: 1.1.1
* Python version: 3.7.7
* Operating System: Windows/Anaconda

### Description

Describe what you were trying to get done.

I am trying to build a representative graph of my GTFS network for a particular day. I have selected a particular date and loaded my feed, like this:

```
service_ids = service_ids_by_date[datetime.date(2019, 6, 29)]
feed = ptg.load_feed(path, view={
    'trips.txt': {
        'service_id': service_ids,
    },
})
```

Tell us what happened, what went wrong, and what you expected to happen.

I expected all of my feed data to correspond to the service id I selected. However the I looked at the stop_times and it includes trip_ids that are not part of the service id.

Therefore I think that the load_feed function and the views work differently than I was expecting. It seems like they just filter the specific txt file that is being specified in the views. 

Do you know if there is a way to load a feed that will only load stops that correspond to the trips that are part of the specified service_id?

**Comment by invisiblefunnel** [comment]
Thanks for posting @d3netxer. Based on your description I think this script should reproduce your error. Can you run it and reply with the output? Please also share the GTFS file if you can.

```python3
import partridge as ptg
print(ptg.__version__)

inpath = "caltrain-ca-us.zip"
_, service_ids = ptg.read_busiest_date(inpath)
feed = ptg.load_feed(inpath, view={"trips.txt": {"service_id": service_ids}})

assert service_ids != set(ptg.load_feed(inpath).trips.service_id), "only one service"
assert service_ids == set(feed.trips.service_id), "bad service"
assert set(feed.trips.trip_id) == set(feed.stop_times.trip_id), "bad trips"
assert set(feed.stop_times.stop_id) == set(feed.stops.stop_id), "bad stops"

print("ok")
```

**Comment by d3netxer** [comment]
I did some further debugging and my error was that I was loading two feeds in the same notebook and I was using the same variable names for both feeds. When I was inspecting earlier I was inspecting earlier cells for the first feed and did not know that the variable really contained the 2nd feed, oops! This is why I thought my stop times weren't corresponding to the service types in my view.

After correcting my mistake, everything works great now. I also tried your nice checking script and all of the assertion lines check out. Thanks
